components/layout/Footer.tsx

```diff
@@ -292,10 +292,6 @@ export const Footer = ({}) => {
 
       {/* Bottom */}
       <div className="flex flex-col lg:flex-row w-full lg:items-center lg:justify-between bg-gradient-to-br from-orange-600 via-orange-800 to-orange-900 text-white px-6 py-8 lg:px-12 gap-6">
-        <div className="flex items-center gap-3 whitespace-nowrap">
-          <span>Stay in touch 👉</span>
-          <EmailForm isFooter />
-        </div>
         <div className="flex drop-shadow-sm flex-wrap justify-end gap-6">
```

Comment:
You should remove the unused EmailForm import at the top of the file

Reply:
Done